packages/gravity-ui-web/src/sass/05-components/03-organisms/_list-cards-basic.scss

```diff
@@ -1,31 +1,28 @@
 .grav-c-list-cards-basic {
-  padding: 0;
-  list-style-type: none;
+  @include grav-l-column-list;
 
   > li + li {
     margin-top: $grav-sp-xl;
   }
 
   @media all and (min-width: grav-breakpoint(large)) {
-    display: flex;
-    flex-wrap: wrap;
+    @include grav-l-column-layout(3, $grav-sp-l);
```

Comment:
Should this not be 2 columns instead of 3?

Reply:
whoops, i was playing around proving that they could work with more columns and must have missed setting this back